### ISSUE TITLE
Why using shrinkwrap?

### ISSUE THREAD

**Issue by analog-nico** [nodesecurity/gulp-nsp#34]
It somehow makes sense to me that `gulp-nsp` uses `npm shrinkwrap` and `shrinkydink` – awesome lib btw. But `nsp` is at 2.6.1 and `gulp-nsp` still depends on `2.3.0`.

I didn't open this issue for asking to change anything other than maybe updating `nsp` at your convenience. Instead I am interested to know what let you to decide to choose a fixed version of `nsp` over a looser version as a dependency or even a peer dependency. As a library developer myself I get [mixed opinions from the community](https://github.com/request/request-promise/issues/142) and thus am looking for arguments to make my decisions.

tl;dr: What are your 2 cents about why you chose to use shrinkwrap for `gulp-nsp`? Thanks a lot!

**Comment by nlf** [comment]
we were using it to pin dependencies in a very specific way, as of this moment in time we have stopped. it may come back, it may not. we'll have to evaluate that.
